src/js/contexts/ResponsiveContext/doc.js

```diff
@@ -28,8 +28,9 @@ export const doc = ResponsiveContext => {
 
 export const themeDoc = {
   'global.breakpoints': {
-    description:
-      'The possible breakpoints that could affect border, direction, gap, margin, pad, and round. The default values help to optimize content for mobile, tablet, and computer.',
+    description: `The possible breakpoints that could affect border, 
```

Comment:
why does your line break after 70 characters? The standard for prettier (grommet) is 80 characters.

Reply:
Prettier wasn't breaking the text for some reason and I was getting an eslint error. Let me go back and make sure prettier is working correctly and make a new commit.

Reply:
Any updates on this one?

Reply:
Still trying to fix my format on save.. can't figure out why it's working perfectly fine for everything else except this. Will work through that today.